v0/destinations/ga4/utils.js

```diff
@@ -357,8 +358,37 @@ const getGA4CustomParameters = (message, keys, exclusionFields, payload) => {
 const responseHandler = (destinationResponse, dest) => {
   const message = `[GA4 Response Handler] - Request Processed Successfully`;
   let { status } = destinationResponse;
+  const { response } = destinationResponse;
   if (status === 204) {
     status = 200;
+    // for GA4 debug validation endpoint, status is always 200
+  } else if (status === 200 && isNotNull(response)) {
+    const {
+      description,
+      validationCode,
+      fieldPath
+    } = destinationResponse?.response?.validationMessages[0];
+    if (destinationResponse.response.validationMessages.length === 0) {
```

Comment:
the `response` variable can be taken from the above which is defined in line: 361?

response.validationMessages[0] 
response.validationMessages.length